server/main.lua

```diff
@@ -40,7 +40,7 @@ RegisterNetEvent('qbx_diving:server:sellCoral', function()
     local src = source
     local player = exports.qbx_core:GetPlayer(src)
     local availableCoral = getCoralInInventory(src)
-    if #availableCoral == 0 then
+    if availableCoral == 0 then
```

Comment:
This won't work as available Coral is a table. Can use table.next to check for an empty table

Reply:
6.1 - Basic Functions of https://www.lua.org/manual/5.4/manual.html

In this current case I believe he wants you to do `if not next(availableCoral)` which checks if there are no more key-value pairs to iterate over. He can correct me if I'm mistaken